app/controllers/users_controller.rb

```diff
@@ -25,10 +25,22 @@ def create
   end
   
   def destroy
```

Comment:
It looks like this action just logs the user out if she's logged in. If that's the case, should this method be called "log_out" rather than destroy? Or, is this perhaps just the first part of the method and we're going to build on this to actually destroy a user record (which I'm not sure if we concluded how we want to handle).

Reply:
I just called it `destroy` because that's what it was in the OAth examples. If we did want to be able to delete users, yeah, I think we could just rename this to `logout` or something